test/e2e/autoscale.go

```diff
@@ -289,24 +289,32 @@ func assertScaleDown(ctx *TestContext) {
 }
 
 func numberOfReadyPods(ctx *TestContext) (float64, error) {
-	// SKS name matches that of revision.
+	// Deployment name matches that of revision.
```

Comment:
The deployment has `-deployment` suffixed, at least for smaller names.

Reply:
https://github.com/knative/serving/blob/main/pkg/reconciler/revision/resources/names/names.go#L22

```golang
import knative.dev/serving/pkg/reconciler/revision/resources/names

deployName := names.Deployment(ctx.resources.Revision)
```

Reply:
Oops, thanks!